src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -197,6 +204,14 @@ export default {
 			type: Number,
 			default: 0,
 		},
+
+		/**
+		 * Id of the last read message when the chat was opened
+		 */
+		lastReadMessage: {
```

Comment:
feels a bit overkill having this property on all components.

wondering if we should read it directly from the messages store (the purpose of stores is to give access to commonly used attributes and remove the need to pass them in everywhere)

Reply:
I guess it makes sense to store it somewhere.